ladybug/dt.py

```diff
@@ -220,3 +224,199 @@ def ToString(self):
     def __repr__(self):
         """Return date time as a string."""
         return self.__str__()
+
+
+class Date(date):
+    """Ladybug Date.
+
+    Properties:
+        month
+        day
+        leap_year
+        doy
+    """
+
+    __slots__ = ()
+
+    def __new__(cls, month=1, day=1, leap_year=False):
+        """Create Ladybug Date.
+
+        Args:
+            month: A value for month between 1-12 (Defualt: 1).
+            day: A value for day between 1-31 (Defualt: 1).
+            leap_year: A boolean to indicate if date is for a leap year
+                (Default: False).
+        """
+        year = 2016 if leap_year else 2017
+        try:
+            return date.__new__(cls, year, month, day)
+        except ValueError as e:
+            raise ValueError("{}:\n\t({}/{})(m/d)".format(e, month, day))
+
+    @classmethod
+    def from_dict(cls, data):
+        """Create date from a dictionary.
+
+        Args:
+            data: {
```

Comment:
This will not render correctly in documentation. I remember that we fixed it once but don't remember for which library

You should use something like

```
.. code-block:: python

  data = { this: that}
```
see here: https://www.sphinx-doc.org/en/1.5/markup/code.html#directive-code-block

Reply:
So there are a number of changes that need to be made across all of our documentation and I have put them in a list here:
https://docs.google.com/document/d/1MmKllWNW5B4TvYlDawzQxS_kq5AFXmnWIBf68uCBCA0/edit?usp=sharing

These changes need to happen across nearly all of the main repos and I think it is going to be easier to update an entire repo altogether rather than having half of it updated and the other half not updated.  So I am going to leave this as it is now and someone (possibly me in a couple of months) will go through and update all of the documentation together at some point.

Reply:
Sounds like a plan! I can also add to the list. If you run sphix commands locally you will see a long list of warnings that should be addressed.

Reply:
Also, I put together an issue so we don't forget:
https://github.com/ladybug-tools/ladybug/issues/158